patch

```diff
@@ -0,0 +1,190 @@
+diff --git a/2048-qt.pro b/2048-qt.pro
```

Comment:
if the maintainer accepts your PR and releases a new version, this patch will not be needed